client/src/docs/command-docs.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2019. Beautiful Code BV, Rotterdam, Netherlands
+ * Licensed under GNU GENERAL PUBLIC LICENSE Version 3.
+ */
+
+import * as React from "react"
+import { CardText } from "reactstrap"
+
+import { Command } from "../state/app-state"
+
+export interface ICommandDoc {
+    title: Command,
+    body: JSX.Element
+}
+/*
+    Start = "Start",
+    Stop = "Stop",
+    Terraform = "Terraform",
+
+ */
+export const COMMAND_DOCS: ICommandDoc[] = [
+    {
+        title: Command.AbandonFix,
+        body:
+            <CardText>
+                While you are fixing the island by terraforming (creating land or water),
+                you can decide to abandon the choice you made and start again.
+            </CardText>,
+    },
+    {
+        title: Command.ClaimHexalot,
+        body:
+            <CardText>
+                When you have reserved a hexalot, and fixed the island to make sure it follows
+                the rules, you can claim the hexalot you reserved.
+                It then belongs to you, since it is completely unique and nobody else can claim the same one.
+            </CardText>,
+    },
+    {
+        title: Command.DiscardGenes,
+        body:
+            <CardText>
+                If you don't like what you're seeing during the evolution and you've given up hope for
+                this particular gotchi, you can discard the current set of genes and start from scratch.
+            </CardText>,
+    },
+    {
+        title: Command.Evolve,
+        body:
+            <CardText>
+                You can start evolving either from the starting position hanging up in the tower
+                at the center of your hexalot, or you can ride a bit first and then start evolving.
+                Evolving creates a group of idential galapagotchis which exist in the same space
+                at the same time, but have slightly mutated genes.
+                The slowest are forgotten and the new generation is made by mutating the genes
+                of the current crop of survivors.
+            </CardText>,
+    },
+    {
+        title: Command.Home,
+        body:
+            <CardText>
+                Wherever you are, you can go back to your home position up in the tower in the
```

Comment:
Maybe I'm not understanding, but why doesn't this just say "go back to your home position up in the tower.."? Don't you want to _just_ explain the command?

Reply:
Ok.